services/collectibles/api.go

```diff
@@ -235,6 +203,20 @@ func (api *API) RemoteBurn(ctx context.Context, chainID uint64, contractAddress
 	return tx.Hash().Hex(), nil
 }
 
+func (api *API) EstimateRemoteBurn(ctx context.Context, chainID uint64, contractAddress string, tokenIds []*bigint.BigInt) (uint64, error) {
```

Comment:
Hmm we won't be able to get this in the transactions list on the sender side unless this generates a MultiTransaction (we need a new type like `MultiTransactionDestroy`), since that's not information we can easily fetch from the blockchain (as opposed to the receiver side, where we simply need to look for a transfer to address `0x0`).

Reply:
Sorry @dlipicar, I do not understand. Do you mean EstimateRemoteBurn function? It works correctly. What is the problem?

Reply:
Clarified in chat, it's all fine!